### PR TITLE
Add new column former contact id to the contact listings.

### PR DESCRIPTION
**Pull request by phgross**
bildschirmfoto 2016-09-14 um 17 16 23

https://basecamp.com/2768704/projects/12330648/todos/270859677

### CODE REVIEW

**Comment by phgross** [comment]
@deiferni

File: opengever/contact/models/archivedcontact.py [hunk 1]
```diff
@@ -22,5 +22,7 @@ class ArchivedContact(ArchiveMixin, Base):
 
     description = Column(UnicodeCoercingText)
 
+    former_contact_id = Column('former_contact_id', Integer, unique=True)
```
Comment (deiferni) [inline opengever/contact/models/archivedcontact.py]: i think we don't have to archive the `former_contact_id` as it should never change

File: opengever/contact/upgrades/20160914150923_add_former_contact_id_column/upgrade.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+class AddFormerContactIdColumn(SchemaMigration):
+    """Add former contact id column.
+    """
+
+    def migrate(self):
+        self.op.add_column(
+            'contacts',
+            Column('former_contact_id', Integer, nullable=True, unique=True))
+
+        self.op.add_column(
```
Comment (deiferni) [inline opengever/contact/upgrades/20160914150923_add_former_contact_id_column/upgrade.py]: same here

File: opengever/contact/models/archivedcontact.py (hunk 1 above)
Comment (phgross) [inline opengever/contact/models/archivedcontact.py]: ✅

File: opengever/contact/upgrades/20160914150923_add_former_contact_id_column/upgrade.py (hunk 2 above)
Comment (phgross) [inline opengever/contact/upgrades/20160914150923_add_former_contact_id_column/upgrade.py]: ✅

**Comment by phgross** [comment]
@deiferni updated ...

**Comment by deiferni** [comment]
cool mit dene neue reviews 😄 - approved 👍

**Pull request merged**